types/phplint/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for phplint 2.0
+// Project: https://github.com/wayneashleyberry/phplint
+// Definitions by: Adam Thompson-Sharpe <https://github.com/MysteryBlokHed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 2.8
+import grunt = require('grunt');
```

Comment:
use reference to local type definition
```suggestion
/// <reference types="grunt" />
```